src/reanimated2/layoutReanimation/web/animationsManager.ts

```diff
@@ -0,0 +1,485 @@
+    ? createAnimationWithExistingTransform(initialAnimationName, transform)
+    : initialAnimationName;
+
+  const animationConfig: AnimationConfig = {
+    animationName: animationName,
+    duration: getDurationFromConfig(
+      config,
+      isLayoutTransition,
+      initialAnimationName as AnimationNames
+    ),
+    delay: getDelayFromConfig(config),
+    easing: getEasingFromConfig(config),
+    callback: getCallbackFromConfig(config),
+  };
+
+  switch (animationType) {
+    case LayoutAnimationType.ENTERING:
+      handleEnteringAnimation(element, animationConfig);
+      break;
+    case LayoutAnimationType.LAYOUT:
+      // `transitionData` is cast as defined because it is a result of calculations made inside componentDidUpdate method.
+      // We can reach this piece of code only from componentDidUpdate, therefore this parameter will be defined.
+
+      // @ts-ignore This property exists in SequencedTransition
+      (transitionData as TransitionData).reversed = config.reversed
+        ? // @ts-ignore This property exists in SequencedTransition
+          config.reversed
+        : false;
+
+      handleLayoutTransition(
+        element,
+        animationConfig,
+        transitionData as TransitionData
+      );
+      break;
+    case LayoutAnimationType.EXITING:
+      handleExitingAnimation(element, animationConfig);
+      break;
+  }
+}
+
+export function handleEnteringAnimation(
+  element: HTMLElement,
+  animationConfig: AnimationConfig
+) {
+  const { delay } = animationConfig;
+
+  // If `delay` === 0, value passed to `setTimeout` will be 0. However, `setTimeout` executes after given amount of time, not exactly after that time
+  // Because of that, we have to immediately toggle on the component when the delay is 0.
+  if (delay === 0) {
+    element.style.visibility = 'initial';
+  } else {
+    setTimeout(() => {
+      element.style.visibility = 'initial';
+    }, delay * 1000);
+  }
+
+  setElementAnimation(element, animationConfig);
+}
+
+export function handleLayoutTransition(
+  element: HTMLElement,
+  animationConfig: AnimationConfig,
+  transitionData: TransitionData
+) {
+  const { animationName } = animationConfig;
+
+  let animationType;
+
+  switch (animationName) {
+    case 'LinearTransition':
+      animationType = TransitionType.LINEAR;
+      break;
+    case 'SequencedTransition':
+      animationType = TransitionType.SEQUENCED;
+      break;
+    case 'FadingTransition':
+      animationType = TransitionType.FADING;
+      break;
+    default:
+      animationType = TransitionType.LINEAR;
+      break;
+  }
+
+  animationConfig.animationName = TransitionGenerator(
+    animationType,
+    transitionData
+  );
+
+  setElementAnimation(element, animationConfig);
+}
+
+export function handleExitingAnimation(
+  element: HTMLElement,
+  animationConfig: AnimationConfig
+) {
+  const parent = element.offsetParent;
+  const tmpElement = element.cloneNode() as HTMLElement;
```

Comment:
Can we think of any cool name to represent this element? What about `zombie` or `ghost`? 😄

Reply:
What about `dummyElement`? 😄